test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/Microsoft.Extensions.AI.Abstractions.Tests.csproj

```diff
@@ -10,6 +10,10 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
```

Comment:
The `JsonSerializerIsReflectionEnabledByDefault` feature switch does not work for .NET framework targets, which I found surprising. The explicit condition doubles down on this quirk so that we can get test coverage for reflection-based serialization as well.